packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -104,8 +105,8 @@ export const SliderStyles = css`
         border-radius: calc(var(--corner-radius) * 1px);
     }
     :host(.vertical) {
-        height: 100%;
-        min-height: calc(var(--design-unit) * 60px);
+        height: calc(var(--fast-slider-height) * 1px);
```

Comment:
I like the approach of providing a variable specifically for this purpose and then having that fallback to a calculation based on thumb size. This seems pretty reasonable as a default.